.eslintrc

```diff
@@ -0,0 +1,26 @@
+{
+  "extends": "airbnb-base",
+  "env": {
+    "es6": true,
+    "browser": true,
+    "jasmine": true,
```

Comment:
that is about eslint checks to allow jasmine things like `describe`… will remove it (it remained from my fork, while rebasing it)

tests itself were mostly not affected in this pull requests